src/main/java/org/javarosa/core/model/FormDef.java

```diff
@@ -16,6 +16,25 @@
 
 package org.javarosa.core.model;
 
+import static java.util.Collections.emptyList;
```

Comment:
I think I'm likely the one who messed up this order previously. 😬 My IDE should now enforce the order that's mostly consistent across existing files.